### ISSUE TITLE
Release Download Problem (Expired Download Request)

### ISSUE THREAD

**Issue by notbatman14** [jMonkeyEngine/sdk#118]
Downloading the Release stable Zip or Exe file could cause problem for some new/low budget developer from developing countries. Not everyone might have the same internet connection speed and stability as everybody else here.
I myself have trouble downloading the exe file for the installer, tried multiple times and all failed because of "Expired Download Request".
My average download speed is around 25 to 55 KB/s downloading the latest stable installer of more than 500MB takes at least 4 hours, but the download seems to expire after 2 hours. Sometimes even only lasting less than an hour.
Is that the problem with my connection or is it the file host problem?
As I can see the file are hosted on amazon cloud.

Is it possible to request some people to split the file into multiple parts? (possibly 100MB or 200MB pieces)

Thanks and Best Regards.

**Comment by grizeldi** [comment]
For engine only, all separate jars are available for download using gradle.

**Comment by MeFisto94** [comment]
I tried to reproduce it by downloading the file with a constant 20kiB ( `wget --limit-rate=20k https://github.com/jMonkeyEngine/sdk/releases/download/3.1-stable-FINAL/jmonkeyplatform-windows-x64.exe`). It took over 8 hours but I did not see any problem, so it might be related to your internet connection which crashes after some time or something?

I've never heard of amazon/github limiting such a download time and judging by the forum posts it seems that the time isn't constant (which is also against a limit). The problem is that other hosters also use the amazon cloud internally... 

The problem is we can't officially split it in 100 MB parts because if we add it additionally we have so many releases which would be confusing people and if we would only release parts, the other users would be annoyed.

I also managed to resume the download (`wget -c ...`). Maybe the expired download request is a thing with amazon (because their link is like `X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIA[....]F20170314%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20170314T081525Z&X-Amz-Expires=300`), however when trying to resume the download from the github link, it worked for me (you might get a new amazon link).

So this is more of a github problem because that should not happen. However someone on the hub might be so nice to split it into parts and upload it. The other one having problems downloading was not having those issues for google drive though. I don't know.
